src/ra_server.erl

```diff
@@ -74,7 +74,8 @@
 -type command_type() :: '$usr' | '$ra_query' | '$ra_join' | '$ra_leave' |
                         '$ra_cluster_change' | '$ra_cluster'.
 
--type command_meta() :: #{from := maybe(from())}.
+-type command_meta() :: #{from := maybe(from()),
+                          ts := integer()}.
```

Comment:
Can we have a longer name? `ts` is not that easy to reason about and search in code.